go.mod

```diff
@@ -368,6 +368,7 @@ require (
 )
 
 replace (
+	github.com/trusch/grpc-proxy => github.com/ItalyPaleAle/grpc-proxy v0.0.0-20220622194522-bb83906fa0cb
```

Comment:
Repo is un-maintained and had to fork this

Reply:
I recommend put the repo under dapr org.

Reply:
That's a good call out @daixiang0.

Please remove this line, due to:

1. Code in `trusch/grpc-proxy` used in Dapr is used for tests only and isn't compiled in `daprd`
2. Using unmaintained repositories is allowed and this ain't the only one

Reply:
Just to clarify, the reason for the fork wasn't that the repo was un-maintained.

The code (which was used for tests only, but was used) did not build anymore after updating protobuf. I had to make [changes](https://github.com/ItalyPaleAle/grpc-proxy/commit/bb83906fa0cb7db6d39a2f36daa1cee3c819ea16), and couldn't open a PR upstream because the repo is unmaintained.

Happy to move the repo to the Dapr org or even just copy/paste the code we need in this repo, it that's better.

Reply:
We have most of the code from that repository copied directly into `dapr/dapr` under `pkg/grpc/proxy`, so the correct thing to do is move the test code also to `dapr/dapr`.

Reply:
I have copied the remaining files into this repo.